docs/logs/customizing-the-sdk/Program.cs

```diff
@@ -41,5 +42,12 @@ public static void Main()
         logger.LogInformation("Hello Information");
         logger.LogWarning("Hello Warning");
         logger.LogError("Hello Error");
+
+        // DEMO of IncludeScopes
```

Comment:
Need more realistic example: https://github.com/open-telemetry/opentelemetry-dotnet/blob/9da4ff798ade9172bb4e14dacc6a847b72531975/docs/logs/extending-the-sdk/Program.cs#L62-L66

Reply:
copied the demo from extending-the-sdk